CHANGELOG.md

```diff
@@ -3,6 +3,59 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.1.0 - Obelix [xxxx-xx-xx]
```

Comment:
Leaving this note for myself so I don't forget to update the date before merging this PR

Reply:
Loving the Asterix themed codename for the the release